eth_crowdfund_be/eth_crowdfund_api/src/models/Campaign.py

```diff
@@ -0,0 +1,86 @@
+from marshmallow import fields, Schema 
+import datetime 
+from . import db
+from .Request import RequestSchema
+
+class Campaign(db.Model):
+  """
+  Campaign Model 
+  """
+
+  # table name 
+  __tablename__ = 'campaigns'
+  id = db.Column(db.Integer, primary_key=True)
+  name = db.Column(db.String, nullable=False)
+  description = db.Column(db.Text)
+  image = db.Column(db.String)
+  manager = db.Column(db.String, nullable=False)
+  contributors = db.Column(db.Integer)
+  upvote = db.Column(db.Integer)
+  min_contribution = db.Column(db.Float, nullable=False)
+  address = db.Column(db.String, unique=True, nullable=False)
+  expiration = db.Column(db.DateTime)
+  created_at = db.Column(db.DateTime)
+  updated_at = db.Column(db.DateTime)
+  requests = db.relationship('Request', backref='campaigns')
+
+  # class constructor
+  def __init__(self, data):
```

Comment:
`__init__` is like the `initialize` method in ruby. This is attaching values from the database to a model